package.json

```diff
@@ -9,6 +9,7 @@
     "build:docs": "wca analyze \"packages/tabs/**/*.js\"",
     "build:types": "tsc -p tsconfig.build.types.json",
     "bundlesize": "rollup -c bundlesize/rollup.config.js && bundlesize",
+    "debug": "web-test-runner \"packages/form-core/test/**/*.test.js\" --watch",
```

Comment:
added this so you can always change it to which package you are currently working on...
later I wanna make it a real command so you can do 
```
yarn debug -g form-core
yarn debug -g overlays
...
```

requires some changes in web-test-runner so postponing this until then

Reply:
so now only form-core supported on purpose?

Reply:
right now I edit `form-core` to what I am working on in the `package.json`

Reply:
which is of course a workaround 😅